lua/config/init.lua

```diff
@@ -124,6 +124,20 @@ function M:init(opts)
   require("lsp.manager").init_defaults(supported_languages)
 end
 
+local function deprecation_notice()
+  for lang, entry in pairs(lvim.lang) do
```

Comment:
what happens if `lvim.lang` is nil ?

Reply:
That shouldn't happen because I'm setting it earlier in the `init` function.